scripts/get_raw_sun_data.py

```diff
@@ -0,0 +1,81 @@
+############
+# Pull raw sun data using pvlib
```

Comment:
Tiny tiny suggestion:

```suggestion
# Compute raw sun data using pvlib
```

"Pull" sounds like we're downloading sun data from somewhere.  Or maybe that's just me?!